scania-digital-design-system/tegel
# feat(grid): new grid

## **Describe pull-request**  
Implementation of new grid

## **Issue Linking:**  
_Choose one of the following options_
- **Jira:** [feat/CDEP-3314-new-grid](https://tegel.atlassian.net/browse/CDEP-3314)    

## **How to test**  
1. Go to Storybook
2. Check in Grid New
3. Resize window and/or toggle fluid.

## **Suggested test steps**

## **Additional context**  
No tests or component. We only ship CSS for new grid system.


[CDEP-3314]: https://tegel.atlassian.net/browse/CDEP-3314?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
